# popgen_start

I am an evolutionary biologist that has chosen to investigate evolutionary signatures through the hourglass of molecular biology. I work under the principles of population genetics theory to characterize the footprints of neutral and adaptive evolution alike. I have started my carrier with the ever accessible mtDNA, then leaped to microsatellites and now I am fully into next generation sequencing, or NGS. 

The original purpose of this repository was to add a few code and code snipets - either of my own or as edits from others previously developed (all credits are given) - that I have been utilizing on my analysis. 

I realized I could rather make something more usefull, less selfish, and that would somehow capitalize all the investment I have made through the years on empirical population genetics. It would also bring transparency to my work, as I will be adding future scripts/code too in this repository. Finally, I positive it will also allow me to trace my progress as a researcher and help me to keep track of all the important yet "easy-to-lost-track" scripts. Sounds quite personal, I know, but hey, it is in the internet so now belongs to everybody. 

Here I present you a series of walkthroughs on pop gen analysis which I decided to reluctantly divide by "marker type". The type of DNA marker used in population genetics largely dictates data preparation,analytical pathways, hypothesis, and overall expectations regarding evolutionary processes that have imprinted their signatures on DNA - and this will make everything easier to organize and present. Note that different sequencing/genotyping strategies may be applied to the same "marker type", i.e. Sanger sequencing vs Illumina sequencing of mitochondrial genes, hence my reluctancy. I will see how it works. 

Hope it helps anybody out there:)

IMPORTANT TO READ: Naturally, the limit is not (unfortunately) the sky but rather my comprehension, creativity, experience and the research oportunities I have been given. 

